sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3860,8 +3860,10 @@ SUBROUTINE MISCLN
 
 !Effective layer helicity
             IF (IGET(988)>0) THEN
+             GRID1=spval
              DO J=JSTA,JEND
                DO I=1,IM
+                  IF(LLOW(I,J)<spval.and.LUPP(I,J)<spval)&
```

Comment:
Suggest check ESRH.

Reply:
grid1 is initialized with spval. so if (ESRH==spval) grid1=ESRH still valid.